<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TauLabs</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="mainTW">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="flightTab">
       <attribute name="title">
        <string>Flight</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="targetsLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Select target</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="targetsCB"/>
          </item>
          <item>
           <widget class="QPushButton" name="buildPB">
            <property name="text">
             <string>Build</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cleanPB">
            <property name="text">
             <string>Clean</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="programPB">
            <property name="text">
             <string>Program</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>GCS</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="title">
           <string>Build configuration</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QRadioButton" name="gcsReleaseRB">
             <property name="text">
              <string>Release</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="gcsDebugRB">
             <property name="text">
              <string>Debug</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="gcsBuildPB">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>Build</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="gcsCleanPB">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>Clean</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="toolsTab">
       <attribute name="title">
        <string>Tools install</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="toolsLayout"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="tool_installPB">
            <property name="text">
             <string>Install</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="tool_cleanPB">
            <property name="text">
             <string>Clean</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Big Hammer</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QPushButton" name="buildAllPB">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>Build All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cleanAllPB">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>Clean All</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Special target</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/new/prefix1/sandwich.JPG</normaloff>:/new/prefix1/sandwich.JPG</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="debugOutput">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelProcessPB">
      <property name="text">
       <string>Cancel running process</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTemplate">
    <property name="title">
     <string>Template</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuVerbosity">
     <property name="title">
      <string>Verbosity</string>
     </property>
     <addaction name="action0"/>
     <addaction name="action1"/>
     <addaction name="action2"/>
    </widget>
    <widget class="QMenu" name="menuJobs">
     <property name="title">
      <string>Jobs</string>
     </property>
     <addaction name="action1_2"/>
     <addaction name="action2_2"/>
     <addaction name="action3"/>
     <addaction name="action4"/>
     <addaction name="action5"/>
     <addaction name="action6"/>
     <addaction name="action7"/>
     <addaction name="action8"/>
    </widget>
    <addaction name="menuVerbosity"/>
    <addaction name="menuJobs"/>
   </widget>
   <addaction name="menuTemplate"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="action0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </action>
  <action name="action1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action1_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action2_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="action6">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="action7">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </action>
  <action name="action8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
